dev/com.ibm.ws.jaxb_fat/test-applications/jaxbApp/src/jaxb/web/JAXBTestServlet.java

```diff
@@ -62,9 +63,26 @@ public void testJakartaJaxbAPILoadedFromLiberty() throws Exception {
     }
 
     @Test
-    @SkipForRepeat({ SkipForRepeat.EE9_FEATURES })
+    @SkipForRepeat({ SkipForRepeat.NO_MODIFICATION, "JAXRS", "JAXB-2.3", SkipForRepeat.EE9_FEATURES })
+    public void testJakartaEE10JaxbAPILoadedFromLiberty() throws Exception {
```

Comment:
could these tests be combined and have there be a conditional for the right String based off of the repeat action?